src/app/components/common/text_output.vue

```diff
@@ -46,14 +46,14 @@ module.exports = {
 <style lang="scss" scoped>
 .text-container {
     background-color: #eeeeee;
-    overflow-y: auto;
-    overflow-x: hidden;
+    overflow: auto;
     height: 200px;
     border-bottom-style: solid;
     border-bottom-width: 1px;
     border-color: #e2e2e2;
     pre {
-        overflow: hidden;
+        width: fit-content;
```

Comment:
this won't work in IE and Edge, so next line is fallback for them

Reply:
Not really needed, as electron runs on chrome but it's a nice touch